algorithms/CPlusPlus/Strings/print-alphabets.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+Description: A C++ program to print first letter of every word in a string
+*/
+
+
+//importing c++ libraries
+#include <iostream>
+#include <string>
+using namespace std;
+
+//function
+string firstAlphabet(string s){
+  //declaring an output string 'o'
+string o;
+//inserting 1st letter in the output string if its not space
+if(s[0]!= ' '){
+   o+=s[0];  
+}
+	 
+    //iterating over the string from index 1
+    //if space is found, then we will keep checking if there are mulitple spaces between 2 words
```

Comment:
minor typo detected by codespell
```suggestion
    //if space is found, then we will keep checking if there are multiple spaces between 2 words
```

Reply:
Hi, I made the following changes
1. fixed typo error
2. fixed indentation